packages/roosterjs-editor-dom/lib/edit/adjustInsertPosition.ts

```diff
@@ -281,6 +282,42 @@ function adjustInsertPositionForNotEditableNode(
     return position;
 }
 
+/**
+ * Adjust the position of a table to be one line after another table.
+ */
+function adjustInsertPositionForTable(
+    root: HTMLElement,
+    nodeToInsert: Node,
+    position: NodePosition,
+    range: Range
+): NodePosition {
+    if (
+        (nodeToInsert.childNodes.length == 1 &&
+            getTagOfNode(nodeToInsert.childNodes[0]) == 'TABLE') ||
+        getTagOfNode(nodeToInsert) == 'TABLE'
+    ) {
+        const { element, offset } = position;
+        const previousElementSibling = element.previousElementSibling;
+        const prevElement =
+            offset - 1 >= 0
+                ? element.childNodes.item(offset - 1)
+                : previousElementSibling?.childNodes.item(
+                      previousElementSibling.childElementCount - 1
+                  );
+        if (prevElement && getTagOfNode(prevElement) == 'TABLE' && element?.isContentEditable) {
```

Comment:
If feel even element is not editable, we should also add this empty line. Any reason we check content editable here?

Reply:
No reason at all, will remove it.